extensions/pyRevitCore.extension/pyRevit.tab/pyRevit.panel/Settings.smartbutton/script.py

```diff
@@ -389,42 +380,23 @@ def _setup_addinfiles(self):
             {str(x.Product.ProductYear):x
              for x in PyRevit.PyRevitAttachments.GetAttachments()}
 
-        for rvt_ver, checkbox in self._addinfiles_cboxes.items():
-            if rvt_ver in attachments:
-                if rvt_ver != HOST_APP.version:
-                    checkbox.Content = \
-                        self._make_product_name(
-                            attachments[rvt_ver].Product,
-                            ''
-                            )
-                else:
-                    checkbox.Content = \
-                        self._make_product_name(
-                            attachments[rvt_ver].Product,
-                            self.get_locale_string("RevitAttachment.Current")
-                            )
-
-                checkbox.IsChecked = True
-                if attachments[rvt_ver].AttachmentType == \
-                        PyRevit.PyRevitAttachmentType.AllUsers:
-                    checkbox.IsEnabled = False
-                    checkbox.Content += self.get_locale_string("RevitAttachment.AllUsers")
-                else:
-                    checkbox.IsEnabled = True
-            else:
-                if rvt_ver in installed_revits:
-                    checkbox.Content = \
-                        self._make_product_name(
-                            installed_revits[rvt_ver],
-                            self.get_locale_string("RevitAttachment.NotAttached")
-                            )
-                    checkbox.IsEnabled = True
-                    checkbox.IsChecked = False
-                else:
-                    checkbox.Content = \
-                        self.get_locale_string("RevitAttachment.NotInstalled").format(rvt_ver)
-                    checkbox.IsEnabled = False
-                    checkbox.IsChecked = False
+        for checkbox  in self.supported_revit_versions_CB:
+            #checkbox.IsEnabled and checkbox.IsChecked are False by default
+            if checkbox.Version not in installed_revits:
+                checkbox.Content += self.get_locale_string("RevitAttachment.NotInstalled") #Change in local string file to avoid passing the version! already in RevitVersionCB class
+                continue
+            product = installed_revits[checkbox.Version]
+            checkbox.Content += ' | {} | {}({}) '.format(product.Name,product.BuildNumber,product.BuildTarget)
```

Comment:
spaces after commas
```suggestion
            checkbox.Content += ' | {} | {}({}) '.format(product.Name, product.BuildNumber, product.BuildTarget)
```

in general, run `pipenv run black extensions/pyRevitCore.extension/pyRevit.tab/pyRevit.panel/Settings.smartbutton/script.py` to use a good formatting standard